bitrise.yml

```diff
@@ -38,37 +39,56 @@ workflows:
     - script@1:
         title: Add default web browser entitlement for Fennec
         inputs:
-        - content: |-
```

Comment:
@dnarcese lets revert this and i'll show you much easier way to make a build in BR :)